src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingleadingasterisk/InputMissingLeadingAsteriskFalsePositive.java

```diff
@@ -0,0 +1,9 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingleadingasterisk;
+
+public class InputMissingLeadingAsteriskFalsePositive {
```

Comment:
why it is false positive?

Reply:
My bad. This is a positive test case. I'll change the name.

Reply:
Done